packages/shared/src/components/cards/common/ShowMoreContent.tsx

```diff
@@ -39,7 +39,7 @@ export default function ShowMoreContent({
   return (
     <div className={className}>
       <p
-        className="select-text break-words typo-body"
+        className="select-text break-words font-serif typo-markdown"
```

Comment:
why serif?

Reply:
That's pretty much the new change in reading experience for the content on post page/modal.

See design: https://www.figma.com/design/C7n8EiXBwV1sYIEHkQHS8R/daily.dev---Design-System?node-id=5133-18302&p=f&m=dev

Reply:
Damn, it looks really weird to me

Reply:
🙈